stack.yaml

```diff
@@ -1,7 +1,7 @@
 # For more information, see: http://docs.haskellstack.org/en/stable/yaml_configuration.html
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-11.4
+resolver: lts-13.16
```

Comment:
Could you make this a recent version of `nightly`?

That way, you should be able to remove all the `servant-*` `extra-deps` below.

Reply:
I'm trying out `nightly-2019-04-05` right now.